mta/mta_services.go

```diff
@@ -304,6 +304,15 @@ func GetBuildParameters(path string, extensions []string) (*ProjectBuild, []stri
 	return mta.BuildParams, messages, nil
 }
 
+//getGlobalParameters - gets the MTA build parameters.
```

Comment:
please update the description:
//GetGlobalParameters - gets the MTA global parameters

Reply:
done